src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -511,26 +511,30 @@ def update_rem_if_mgmt(self, if_oid, if_name):
                 return
             time_mark = int(lldp_kvs[b'lldp_rem_time_mark'])
             remote_index = int(lldp_kvs[b'lldp_rem_index'])
-            subtype = self.get_subtype(mgmt_ip_str)
-            ip_hex = self.get_ip_hex(mgmt_ip_str, subtype)
-            if subtype == ManAddrConst.man_addr_subtype_ipv4:
-                addr_subtype_sub_oid = 4
-                mgmt_ip_sub_oid = (addr_subtype_sub_oid, *[int(i) for i in mgmt_ip_str.split('.')])
-            elif subtype == ManAddrConst.man_addr_subtype_ipv6:
-                addr_subtype_sub_oid = 6
-                mgmt_ip_sub_oid = (addr_subtype_sub_oid, *[int(i, 16) if i else 0 for i in mgmt_ip_str.split(':')])
-            else:
-                logger.warning("Invalid management IP {}".format(mgmt_ip_str))
-                return
-            self.if_range.append((time_mark,
-                                  if_oid,
-                                  remote_index,
-                                  subtype,
-                                  *mgmt_ip_sub_oid))
-
-            self.mgmt_ips.update({if_name: {"ip_str": mgmt_ip_str,
-                                            "addr_subtype": subtype,
-                                            "addr_hex": ip_hex}})
+
+            for mgmt_ip in mgmt_ip_str.split(','):
+                ipa  = ipaddress.ip_address(mgmt_ip)
+                subtype = self.get_subtype(mgmt_ip)
+                ip_hex = self.get_ip_hex(mgmt_ip, subtype)
+
+                if subtype == ManAddrConst.man_addr_subtype_ipv4:
+                    addr_subtype_sub_oid = 4
+                    mgmt_ip_sub_oid = (addr_subtype_sub_oid,) + tuple(i for i in ipa.packed)
+                elif subtype == ManAddrConst.man_addr_subtype_ipv6:
+
+                    addr_subtype_sub_oid = 16
+                    mgmt_ip_sub_oid = (addr_subtype_sub_oid,) + tuple(i for i in ipa.packed)
```

Comment:
>mgmt_ip_sub_oid = (addr_subtype_sub_oid,) + tuple(i for i in ipa.packed) [](start = 20, length = 72)

It works for both ipv4 and ipv6. You may move it outside.

Reply:
Modified as requested.